src/Sulu/Bundle/MediaBundle/Content/Types/MediaSelectionContentType.php

```diff
@@ -249,4 +255,25 @@ public function mapPropertyMetadata(ContentPropertyMetadata $propertyMetadata):
 
         return new PropertyMetadata($propertyMetadata->getName(), $mandatory, $mediaSelectionMetadata);
     }
+
+    public function getReferences(PropertyInterface $property, PropertyValue $propertyValue, ReferenceCollector $referenceCollector): void
```

Comment:
check if propertyValue is required here

Reply:
added the value to the `property` and removed the `PropertyValue`